frontend/packages/dev-console/src/utils/create-knative-utils.ts

```diff
@@ -2,13 +2,15 @@ import { k8sCreate, K8sResourceKind } from '@console/internal/module/k8s';
 import { KsServiceModel } from '../models';
 
 export const createKnativeService = (
-  name: string,
-  namespace: string,
-  imageStreamName: string,
-  imageStreamTag: string,
+  formData,
   cpuResource: string = '100m',
   memoryResource: string = '100Mi',
 ): Promise<K8sResourceKind> => {
+  const {
+    project: { name: namespace },
+    name,
+    isi: { name: isiName, tag },
+  } = formData;
```

Comment:
I think you should revert this so that we can use this function for git import too. If we keep this change then it only works with deploy image.

Reply:
yes, I did this just to demonstrate. It's in the raw state.

Need to refine